css-layout-classes/styles.css

```diff
@@ -0,0 +1,25 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.img {
+  max-width: 100%;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-one-fourth {
+  flex-basis: calc(100% / 4);
+}
+
+.column-two-thirds {
+  flex-basis: calc(100% / 1);
+}
```

Comment:
You should have a selector here for `column-full`. In this case it works without it, but for semantics, and to follow the same structure, we should still create the ruleset for it.